core/win32/syscallx.h

```diff
@@ -94,66 +94,69 @@
+SYSCALL(FlushInstructionCache,        true,     3, 0x0c, 0x36, 0x36, 0x36, 0x42, 0x04e, 0x0c, 0x098, 0x052, 0x08d, 0x0c1, 0x08d, 0x0bf, 0x07d,0x0c2, 0x11e,0xc00d4,0x0d4, 0x122,0xc00d6,0x0d6, 0x127,0xc00d9,0x0d9, 0x129,0xc00da,0x0da, 0x12b,0xc00dc,0x0dc, 0x12e,0xc00df,0x0df, 0x131,0xc00e0,0x0e0, 0x131,0xc00e1,0x0e1, 0x131,0xc00e2,0x0e2)
+SYSCALL(FreeUserPhysicalPages,        true,     3, 0x0c, NONE, NONE, NONE, 0x46, 0x052,    0, 0x09c, 0x056, 0x092, 0x0c6, 0x092, 0x0c4, 0x082,0x0c7, 0x119,0x000d9,0x0d9, 0x11d,0x000db,0x0db, 0x122,0x000de,0x0de, 0x124,0x000df,0x0df, 0x126,0x000e1,0x0e1, 0x129,0x000e4,0x0e4, 0x12c,0x000e5,0x0e5, 0x12c,0x000e6,0x0e6, 0x12c,0x000e7,0x0e7)
+SYSCALL(MapUserPhysicalPages,         true,     3, 0x0c, NONE, NONE, NONE, 0x5b, 0x06a, 0x0a, 0x0b2, 0x06f, 0x0af, 0x0e7, 0x0af, 0x0e4, 0x0a6,0x0e7, 0x0f5,0xa00f9,0x0f9, 0x0f8,0xa00fc,0x0fc, 0x0fc,0xa0101,0x101, 0x0fc,0xa0104,0x104, 0x0fe,0xa0106,0x106, 0x100,0xa010a,0x10a, 0x103,0xa010b,0x10b, 0x103,0xa010c,0x10c, 0x103,0xa010d,0x10d)
+SYSCALL(SetInformationVirtualMemory,  true,     6, 0x18, NONE, NONE, NONE, NONE,  NONE,    0,  NONE,  NONE,  NONE,  NONE,  NONE,  NONE,  NONE, NONE, 0x0c9,0x00177,0x177, 0x0cc,0x0017a,0x17a, 0x0ce,0x00182,0x182, 0x0ce,0x00185,0x185, 0x0d0,0x0018b,0x18b, 0x0d2,0x00191,0x191, 0x0d4,0x00194,0x194, 0x0d4,0x00196,0x196, 0x0d4,0x00197,0x197)
+SYSCALL(Wow64AllocateVirtualMemory64, true,     7, 0x1c, NONE, NONE, NONE, NONE,  NONE,    0,  NONE,  NONE,  NONE,  NONE,  NONE,  NONE,  NONE, NONE,  NONE,0x001bb, NONE,  NONE,0x001bf, NONE,  NONE,0x001c7, NONE,  NONE,0x001ca, NONE,  NONE,0x001d0, NONE,  NONE,0x001d6, NONE,  NONE,0x001da, NONE,  NONE,0x001dc, NONE,  NONE,    UNK, NONE)
+SYSCALL(OpenSection,                  true,     3, 0x0c, 0x56, 0x56, 0x56, 0x6c, 0x07d,    0, 0x034, 0x083, 0x0c5, 0x034, 0x0c5, 0x034, 0x0c2,0x034, 0x0d9,0x00035,0x035, 0x0dc,0x00036,0x036, 0x0de,0x00037,0x037, 0x0de,0x00037,0x037, 0x0e0,0x00037,0x037, 0x0e2,0x00037,0x037, 0x0e4,0x00037,0x037, 0x0e4,0x00037,0x037, 0x0e4,0x00037,0x037)
+SYSCALL(CreateSection,                true,     7, 0x1c, 0x21, 0x21, 0x21, 0x2b, 0x032,    0, 0x047, 0x034, 0x04b, 0x047, 0x04b, 0x047, 0x054,0x047, 0x150,0x00048,0x048, 0x154,0x00049,0x049, 0x15a,0x0004a,0x04a, 0x15c,0x0004a,0x04a, 0x15e,0x0004a,0x04a, 0x161,0x0004a,0x04a, 0x164,0x0004a,0x04a, 0x164,0x0004a,0x04a, 0x164,0x0004a,0x04a)
+SYSCALL(CreateSectionEx,              true,     9, 0x24, NONE, NONE, NONE, NONE,  NONE,    0,  NONE,  NONE,  NONE,  NONE,  NONE,  NONE,  NONE, NONE,  NONE,   NONE, NONE,  NONE,   NONE, NONE,  NONE,   NONE, NONE,  NONE,   NONE, NONE,  NONE,   NONE, NONE,  NONE,   NONE, NONE,  NONE,   NONE, NONE,  NONE,   NONE, NONE,   UNK,0x000b9,0x0b9)
```

Comment:
Here is my list extracted from my 32-bit win10 1809 VM.  I see your CreateSection does not match mine -- yours is the Ex.  Please cross-check the others.

```
NtContinue                               0x00000181,
NtCallbackReturn                         0x00000195,
NtSetContextThread                       0x00000060,
NtGetContextThread                       0x00000124,
NtCreateProcess                          0x00000169,
NtCreateProcessEx                        0x00000168,
NtCreateUserProcess                      0x00000158,
NtTerminateProcess                       0x00000024,
NtCreateThread                           0x00000160,
NtCreateThreadEx                         0x0000015f,
NtCreateWorkerFactory                    0x00000154,
NtTerminateThread                        0x00000023,
NtSuspendThread                          0x00000028,
NtResumeThread                           0x0000006f,
NtQueryInformationThread                 0x000000b7,
NtAllocateVirtualMemory                  0x000001b1,
NtAllocateVirtualMemoryEx                0x000001cc,
NtFreeVirtualMemory                      0x0000012a,
NtProtectVirtualMemory                   0x000000ce,
NtQueryVirtualMemory                     0x00000097,
NtWriteVirtualMemory                     0x00000004,
NtMapViewOfSection                       0x00000101,
NtMapViewOfSectionEx                     0x000001cd,
NtUnmapViewOfSection                     0x00000014,
NtUnmapViewOfSectionEx                   0x00000015,
NtFlushInstructionCache                  0x00000130,
NtFreeUserPhysicalPages                  0x0000012b,
NtMapUserPhysicalPages                   0x00000103,
NtSetInformationVirtualMemory            0x000000d4,
Wow64AllocateVirtualMemory64                   NONE,
NtOpenSection                            0x000000e4,
NtCreateSection                          0x00000163,
NtCreateSectionEx                        0x00000164,
NtClose                                  0x0000018d,
NtDuplicateObject                        0x00000141,
NtAlertResumeThread                      0x000001b8,
NtOpenFile                               0x000000f4,
NtTestAlert                              0x00000022,
NtRaiseException                         0x00000090,
NtCreateFile                             0x00000178,
NtCreateKey                              0x00000172,
NtOpenKey                                0x000000f1,
NtOpenKeyEx                              0x000000ef,
NtOpenProcess                            0x000000e9,
NtOpenProcessToken                       0x000000e8,
NtOpenProcessTokenEx                     0x000000e7,
NtOpenThread                             0x000000e0,
NtOpenThreadToken                        0x000000df,
NtOpenThreadTokenEx                      0x000000de,
NtQueryAttributesFile                    0x000000cc,
NtQueryFullAttributesFile                0x000000c0,
NtSetInformationFile                     0x00000053,
NtSetInformationThread                   0x0000004d,
NtSetValueKey                            0x00000033,
NtTerminateJobObject                     0x00000025,
NtAlertThread                            0x000001b7,
NtDelayExecution                         0x00000150,
NtReplyWaitReceivePort                   0x00000078,
NtReplyWaitReceivePortEx                 0x00000077,
NtReplyWaitReplyPort                     0x00000076,
ReplyWaitSendChannel                           NONE,
NtRequestWaitReplyPort                   0x00000074,
SendWaitReplyChannel                           NONE,
NtSetHighWaitLowEventPair                0x00000056,
NtSetLowWaitHighEventPair                0x00000040,
NtSignalAndWaitForSingleObject           0x0000002e,
NtWaitForDebugEvent                      0x0000000f,
NtWaitForKeyedEvent                      0x0000000e,
NtWaitForMultipleObjects                 0x0000000d,
NtWaitForSingleObject                    0x0000000b,
NtWaitHighEventPair                      0x00000009,
NtWaitLowEventPair                       0x00000008,
```